src/tests/multihost/basic/conftest.py

```diff
@@ -89,6 +89,8 @@ def setup_kerberos(session_multihost, request):
     def remove_kerberos():
         """ Remove kerberos instance """
         krb.destroy_krb5server()
+        remove_keytab = 'rm -f /etc/krb5.keytab'
+        session_multihost.master[0].run_command(remove_keytab)
```

Comment:
This also seems like something that should be in a separate commit

Reply:
okay